### PR TITLE
Move netdb.h and ioctl.h into wolfio.c

### PR DESCRIPTION
**Pull request by bandi13**
Better way to fix exclusion of netdb.h in the header files.

### CODE REVIEW

**Comment by bandi13** [comment]
retest this please

**Comment by dgarske** [comment]
FYI: This breaks wolfTPM again. The right solution for wolfTPM is to handle inclusion of netdb.h on its own. I'll put up another PR for that.

**Comment by dgarske** [comment]
> FYI: This breaks wolfTPM again. The right solution for wolfTPM is to handle inclusion of netdb.h on its own. I'll put up another PR for that.

https://github.com/wolfSSL/wolfTPM/pull/333

**Review by douzzer — APPROVED** [review]

File: wolfssl/wolfio.h [hunk 1]
```diff
@@ -163,15 +161,9 @@
             #include <sys/socket.h>
             #include <arpa/inet.h>
             #include <netinet/in.h>
-            #ifdef HAVE_NETDB_H
```
Comment (farazrbx) [inline wolfssl/wolfio.h:166]: Why`HAVE_NETDB_H` and `HAVE_SYS_IOCTL_H` is removed? :cry:

Comment (dgarske) [reply wolfssl/wolfio.h:166]: See #7381

**Pull request merged**